crates/node/src/service/rpc.rs

```diff
@@ -46,8 +46,8 @@ impl RpcService {
                 (true, false)
             }
         };
-        let (read, write, trace) = (rpcs, rpcs, rpcs);
-        let starknet = Starknet::new(Arc::clone(db.backend()), chain_config.clone(), add_txs_method_provider);
+        let (read, write, trace, internal) = (rpcs, rpcs, rpcs, node_operator);
```

Comment:
It will eventually be necessary to secure the origin of internal calls (verification of origin, bind on a specific port, etc.)

Reply:
ideally internal call shouldn't be public, so yeah I agree, shall we cover this in this PR tho?